pkg/engine/apicall/apiCall.go

```diff
@@ -81,9 +81,9 @@ func (a *apiCall) Execute(ctx context.Context, call *kyvernov1.APICall) ([]byte,
 	return a.executor.Execute(ctx, call)
 }
 
-func (a *apiCall) transformAndStore(jsonData []byte, invalid ...bool) ([]byte, error) {
+func (a *apiCall) transformAndStore(jsonData []byte, invalid bool) ([]byte, error) {
 	//If the urlPath is invalid then JMESPath should be ""
-	if invalid[0] {
+	if invalid {
```

Comment:
If the URL is invalid, should the jmespath even be considered? I think it will throw an error before applying the jmespath

Reply:
Yes, when url is invalid jmespath should be ignored that is why i have made 
```go
a.entry.APICall.JMESPath == ""
```

Reply:
But i think 
```go
if a.entry.APICall.JMESPath == "" || invalid {
		err := a.jsonCtx.AddContextEntry(a.entry.Name, jsonData)
		if err != nil {
			return nil, fmt.Errorf("failed to add resource data to context entry %s: %w", a.entry.Name, err)
		}

		return jsonData, nil
	}
```
will be more appropriate